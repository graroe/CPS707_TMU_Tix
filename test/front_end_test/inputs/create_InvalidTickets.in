login
admin
create
CirqueDuSoliel
20231027
10000
ten
300.5
-1
!q
logout