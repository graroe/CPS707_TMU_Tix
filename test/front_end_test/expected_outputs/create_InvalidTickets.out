Enter session type <sales or admin>: 
Admin login successful.
Enter new event name <max 15 chars>: 
Enter date of event <format YYYYMMDD>: 
Enter number of available tickets <max 9999>: 
Cannot exceed 9999 tickets.\nEnter number of available tickets <max 9999>: 
Enter number of available tickets <max 9999>: 
Enter number of available tickets <max 9999>: 
Enter number of available tickets <max 9999>: 
Transaction canceled
Session terminated.